css/styles.css

```diff
@@ -128,3 +136,68 @@ ul {
     color: var(--Grey);
     padding-bottom: 20px;  
 }
+
+/* Search module */
+
+.logo {
+    text-align: center;
+    margin: 0 auto;
+    padding-top: 110px;
+}
+
+.logo img {
+    width: 250px;
+}
+
+.search {
+    text-align: center;
+    padding: 12px; 
+}
+
+.search input {
+   border: none;
+   width: 450px;
+}
+
+.search-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 550px;
+    height: 40px;
+    border-radius: 20px;
+    border: 1px solid var(--Alto);
+    margin: 0 auto;
+}
+
+.search-icon-right {
+    display: flex;
+    align-items: center;
+    padding: 0 10px;
+    gap: 10px;
+}
+
+.search-icon-left {
+    display: flex;
+    align-items: center;
+    padding: 0 10px;
+}
+
+.google-buttons {
+   text-align: center;
+   
+}
+
+.google-buttons__search {
```

Comment:
Please refactor this code so we use `.button` class for both buttons (in header and search). Apply different styling using _BEM_ modifiers.